### PR TITLE
add release notes for v1.1.10

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/releases/release-1.1.10.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- TidbMonitor 支持 `remotewrite` ([#3679](https://github.com/pingcap/tidb-operator/pull/3679), [@mikechengwei](https://github.com/mikechengwei))
+- 支持为 TiDB 集群各组件配置 init containers ([#3713](https://github.com/pingcap/tidb-operator/pull/3713), [@handlerww](https://github.com/handlerww))
+- TiDB Lightning chart 支持 local backend ([#3644](https://github.com/pingcap/tidb-operator/pull/3644), [@csuzhangxc](https://github.com/csuzhangxc))
+
+## 优化提升
+
+- 支持为 TiDB slow log 自定义存储 ([#3731](https://github.com/pingcap/tidb-operator/pull/3731), [@BinChenn](https://github.com/BinChenn))
+- 为 TidbMonitor 中的 scrape jobs 增加 `tidb_cluster` label 以支持多集群监控 ([#3750](https://github.com/pingcap/tidb-operator/pull/3750), [@mikechengwei](https://github.com/mikechengwei))
+- 升级过程中，等待 TiKV Pod 重建后再删除对应的 evict leader scheduler ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
+- 优化自定义资源同步间隔 ([#3700](https://github.com/pingcap/tidb-operator/pull/3700), [@dragonly](https://github.com/dragonly))
```
Comment (dragonly) [inline zh/releases/release-1.1.10.md:27]:
```suggestion
- 优化自定义资源 (CR) 同步间隔 ([#3700](https://github.com/pingcap/tidb-operator/pull/3700), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.1.10.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Add local backend support to the TiDB Lightning chart ([#3644](https://github.com/pingcap/tidb-operator/pull/3644), [@csuzhangxc](https://github.com/csuzhangxc))
+
+## Improvements
+
+- Support customizing the storage config for TiDB slow log ([#3731](https://github.com/pingcap/tidb-operator/pull/3731), [@BinChenn](https://github.com/BinChenn))
+- Add `tidb_cluster` label for the scrape jobs in TidbMonitor to support multiple clusters monitoring ([#3750](https://github.com/pingcap/tidb-operator/pull/3750), [@mikechengwei](https://github.com/mikechengwei))
+- Delete evict leader scheduler after TiKV Pod is recreated during upgrade ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
+- Optimize CR syncing intervals ([#3700](https://github.com/pingcap/tidb-operator/pull/3700), [@dragonly](https://github.com/dragonly))
+- Supports to persist checkpoint for TiDB Lightning helm chart ([#3653](https://github.com/pingcap/tidb-operator/pull/3653), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (dragonly) [inline en/releases/release-1.1.10.md:28]:
```suggestion
- Support persisting checkpoint for TiDB Lightning helm chart ([#3653](https://github.com/pingcap/tidb-operator/pull/3653), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.1.10.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- TidbMonitor support `remotewrite` configuration ([#3679](https://github.com/pingcap/tidb-operator/pull/3679), [@mikechengwei](https://github.com/mikechengwei))
+- Support configuring init containers for components in TiDB cluster ([#3713](https://github.com/pingcap/tidb-operator/pull/3713), [@handlerww](https://github.com/handlerww))
+- Add local backend support to the TiDB Lightning chart ([#3644](https://github.com/pingcap/tidb-operator/pull/3644), [@csuzhangxc](https://github.com/csuzhangxc))
+
+## Improvements
+
+- Support customizing the storage config for TiDB slow log ([#3731](https://github.com/pingcap/tidb-operator/pull/3731), [@BinChenn](https://github.com/BinChenn))
+- Add `tidb_cluster` label for the scrape jobs in TidbMonitor to support multiple clusters monitoring ([#3750](https://github.com/pingcap/tidb-operator/pull/3750), [@mikechengwei](https://github.com/mikechengwei))
+- Delete evict leader scheduler after TiKV Pod is recreated during upgrade ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
```
Comment (dragonly) [inline en/releases/release-1.1.10.md:26]:
```suggestion
- Delete evict leader scheduler after TiKV Pod is recreated during upgrade to reduce performance jitter ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
```

File: en/releases/release-1.1.10.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- Optimize CR syncing intervals ([#3700](https://github.com/pingcap/tidb-operator/pull/3700), [@dragonly](https://github.com/dragonly))
+- Supports to persist checkpoint for TiDB Lightning helm chart ([#3653](https://github.com/pingcap/tidb-operator/pull/3653), [@csuzhangxc](https://github.com/csuzhangxc))
+- Change the directory to save the customized alert rules in TidbMonitor from `tidb:${tidb_image_version}` to `tidb:${initializer_image_version}`. Please note that if the `spec.initializer.version` in the TidbMonitor does not match with the TiDB version in the TidbCluster, upgrade TiDB Operator will cause the re-creation of the monitor Pod ([#3684](https://github.com/pingcap/tidb-operator/pull/3684), [@BinChenn](https://github.com/BinChenn))
```
Comment (dragonly) [inline en/releases/release-1.1.10.md:29]:
```suggestion
- Change the directory from `tidb:${tidb_image_version}` to `tidb:${initializer_image_version}` to save the customized alert rules in `TidbMonitor`. Please note that if the `spec.initializer.version` in `TidbMonitor` does not match the TiDB version in `TidbCluster`, then upgrading TiDB Operator will cause the re-creation of the monitor Pod ([#3684](https://github.com/pingcap/tidb-operator/pull/3684), [@BinChenn](https://github.com/BinChenn))
```

File: en/releases/release-1.1.10.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Bug Fixes
+
+- Fix the issue that backup and restore jobs with BR fail without configuring `spec.from` or `spec.to` when TLS is enabled for the TiDB cluster ([#3707](https://github.com/pingcap/tidb-operator/pull/3707), [@BinChenn](https://github.com/BinChenn))
```
Comment (dragonly) [inline en/releases/release-1.1.10.md:33]: This message is confusing, could you please make it simpler to understand?

File: en/releases/release-1.1.10.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+- Fix the bug that if advanced StatefulSet is enabled and `delete-slots` annotations is added for PD or TiKV, the Pods whose ordinal bigger than `replicas - 1` will be terminated directly without any pre-delete operations such as evicting leaders ([#3702](https://github.com/pingcap/tidb-operator/pull/3702), [@cvvz](https://github.com/cvvz))
```
Comment (dragonly) [inline en/releases/release-1.1.10.md:34]:
```suggestion
- Fix the bug that if advanced StatefulSet is enabled and `delete-slots` annotations is added for PD or TiKV, the Pods with ordinal greater than `replicas - 1` will be terminated directly without any pre-delete operations such as evicting leaders ([#3702](https://github.com/pingcap/tidb-operator/pull/3702), [@cvvz](https://github.com/cvvz))
```

File: en/releases/release-1.1.10.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+- Fix the issue that the status of backup or restore is not `Failed` after the pod has been evicted or killed. ([#3696](https://github.com/pingcap/tidb-operator/pull/3696), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (dragonly) [inline en/releases/release-1.1.10.md:35]:
```suggestion
- Fix the issue that the status of backup and restore is not `Failed` after the pod has been evicted or killed. ([#3696](https://github.com/pingcap/tidb-operator/pull/3696), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.1.10.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+- Fix the issue that when TiKV cluster is not bootstrapped, the following updates of TiKV component are not able to be synced ([#3694](https://github.com/pingcap/tidb-operator/pull/3694), [@cvvz](https://github.com/cvvz))
```
Comment (dragonly) [inline en/releases/release-1.1.10.md:36]:
```suggestion
- Fix the issue that when TiKV cluster is not bootstrapped, the TiKV component could not be recovered by editing `TidbCluster` CR ([#3694](https://github.com/pingcap/tidb-operator/pull/3694), [@cvvz](https://github.com/cvvz))
```

File: en/releases/release-1.1.10.md (hunk 4 above)
Comment (ran-huang) [inline en/releases/release-1.1.10.md:29]: Duplicate `Rolling Update Changes`?

File: en/releases/release-1.1.10.md (hunk 5 above)
Comment (ran-huang) [inline en/releases/release-1.1.10.md:33]:
```suggestion
- Fix the issue that when TLS is enabled for the TiDB cluster, if `spec.from` or `spec.to` is not configured, backup and restore jobs with BR might fail ([#3707](https://github.com/pingcap/tidb-operator/pull/3707), [@BinChenn](https://github.com/BinChenn))
```

File: zh/releases/release-1.1.10.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+- TiDB Lightning chart 支持持久化 checkpoint ([#3653](https://github.com/pingcap/tidb-operator/pull/3653), [@csuzhangxc](https://github.com/csuzhangxc))
+- 将 TidbMonitor 自定义告警规则的存储路径从 `tidb:${tidb_image_version}` 修改为 `tidb:${initializer_image_version}`。请注意，如果 TidbMonitor 中 `spec.initializer.version` 和 TidbCluster 中 TiDB 组件的版本不一致，升级 TiDB Operator 会导致 TidbMonitor Pod 删除重建 ([#3684](https://github.com/pingcap/tidb-operator/pull/3684), [@BinChenn](https://github.com/BinChenn))
+
+## Bug 修复
+
+- 修复在集群开启 TLS 的情况下，如果不配置 `spec.from` 或者 `spec.to`，使用 BR 备份或者恢复会失败的问题 ([#3707](https://github.com/pingcap/tidb-operator/pull/3707), [@BinChenn](https://github.com/BinChenn))
+- 修复在开启advanced StatefulSet 并且为 PD、TiKV 设置了 `delete-slots` annotations 的情况下，序号大于 `replicas - 1` 的 Pod 在升级过程中不会进行迁移 leader 的问题 ([#3702](https://github.com/pingcap/tidb-operator/pull/3702), [@cvvz](https://github.com/cvvz))
```
Comment (ran-huang) [inline zh/releases/release-1.1.10.md:34]: #
```suggestion
- 修复在开启 Advanced StatefulSet 并且为 PD、TiKV 设置了 `delete-slots` annotations 的情况下，序号大于 `replicas - 1` 的 Pod 在升级过程中不会进行迁移 leader 的问题 ([#3702](https://github.com/pingcap/tidb-operator/pull/3702), [@cvvz](https://github.com/cvvz))
```

File: en/releases/release-1.1.10.md (hunk 6 above)
Comment (ran-huang) [inline en/releases/release-1.1.10.md:34]:
```suggestion
- Fix the bug that if advanced StatefulSet is enabled and `delete-slots` annotations are added for PD or TiKV, the Pods whose ordinal is bigger than `replicas - 1` will be terminated directly without any pre-delete operations such as evicting leaders ([#3702](https://github.com/pingcap/tidb-operator/pull/3702), [@cvvz](https://github.com/cvvz))
```

File: en/releases/release-1.1.10.md (hunk 7 above)
Comment (ran-huang) [inline en/releases/release-1.1.10.md:35]:
```suggestion
- Fix the issue that after the Pod has been evicted or killed, the status of backup or restore is not `Failed` ([#3696](https://github.com/pingcap/tidb-operator/pull/3696), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.1.10.md (hunk 8 above)
Comment (ran-huang) [inline en/releases/release-1.1.10.md:36]: The Chinese and English texts seem inconsistent.

**Comment by dragonly** [comment]
There are duplicated review comments, please resolve those related to grammars proposed by @ran-huang first.

**Comment by DanielZhangQD** [comment]
@dragonly @ran-huang PTAL again.

File: en/releases/release-1.1.10.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+- Support canary upgrade of TiDB Operator ([#3548](https://github.com/pingcap/tidb-operator/pull/3548), [@shonge](https://github.com/shonge), [#3554](https://github.com/pingcap/tidb-operator/pull/3554), [@cvvz](https://github.com/cvvz))
+- TidbMonitor supports `remotewrite` configuration ([#3679](https://github.com/pingcap/tidb-operator/pull/3679), [@mikechengwei](https://github.com/mikechengwei))
+- Support configuring init containers for components in the TiDB cluster ([#3713](https://github.com/pingcap/tidb-operator/pull/3713), [@handlerww](https://github.com/handlerww))
+- Add local backend support to the TiDB Lightning chart ([#3644](https://github.com/pingcap/tidb-operator/pull/3644), [@csuzhangxc](https://github.com/csuzhangxc))
+
+## Improvements
+
+- Support customizing the storage config for TiDB slow log ([#3731](https://github.com/pingcap/tidb-operator/pull/3731), [@BinChenn](https://github.com/BinChenn))
+- Add `tidb_cluster` label for the scrape jobs in TidbMonitor to support monitoring multiple clusters ([#3750](https://github.com/pingcap/tidb-operator/pull/3750), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (ran-huang) [inline en/releases/release-1.1.10.md:25]:
```suggestion
- Add the `tidb_cluster` label for the scrape jobs in TidbMonitor to support monitoring multiple clusters ([#3750](https://github.com/pingcap/tidb-operator/pull/3750), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.1.10.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+- Supports persisting checkpoint for TiDB Lightning helm chart ([#3653](https://github.com/pingcap/tidb-operator/pull/3653), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (ran-huang) [inline en/releases/release-1.1.10.md:26]:
```suggestion
- Supports persisting checkpoint for the TiDB Lightning helm chart ([#3653](https://github.com/pingcap/tidb-operator/pull/3653), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.1.10.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+- Change the directory of the customized alert rules in TidbMonitor from `tidb:${tidb_image_version}` to `tidb:${initializer_image_version}` so that when the TiDB cluster is upgraded afterwards, the TidbMonitor Pod will not be recreated ([#3684](https://github.com/pingcap/tidb-operator/pull/3684), [@BinChenn](https://github.com/BinChenn))
+
+## Bug Fixes
+
+- Fix the issue that when TLS is enabled for the TiDB cluster, if `spec.from` or `spec.to` is not configured, backup and restore jobs with BR might fail ([#3707](https://github.com/pingcap/tidb-operator/pull/3707), [@BinChenn](https://github.com/BinChenn))
+- Fix the bug that if advanced StatefulSet is enabled and `delete-slots` annotations are added for PD or TiKV, the Pods whose ordinal is bigger than `replicas - 1` will be terminated directly without any pre-delete operations such as evicting leaders ([#3702](https://github.com/pingcap/tidb-operator/pull/3702), [@cvvz](https://github.com/cvvz))
```
Comment (ran-huang) [inline en/releases/release-1.1.10.md:32]:
```suggestion
- Fix the bug that if the advanced StatefulSet is enabled and `delete-slots` annotations are added for PD or TiKV, the Pods whose ordinal is bigger than `replicas - 1` will be terminated directly without any pre-delete operations such as evicting leaders ([#3702](https://github.com/pingcap/tidb-operator/pull/3702), [@cvvz](https://github.com/cvvz))
```

File: en/releases/release-1.1.10.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+- Fix the issue that after the Pod has been evicted or killed, the status of backup or restore is not updated to `Failed` ([#3696](https://github.com/pingcap/tidb-operator/pull/3696), [@csuzhangxc](https://github.com/csuzhangxc))
+- Fix the issue that when TiKV cluster is not bootstrapped due to incorrect configuration, the TiKV component could not be recovered by editing `TidbCluster` CR ([#3694](https://github.com/pingcap/tidb-operator/pull/3694), [@cvvz](https://github.com/cvvz))
```
Comment (ran-huang) [inline en/releases/release-1.1.10.md:34]:
```suggestion
- Fix the issue that when the TiKV cluster is not bootstrapped due to incorrect configuration, the TiKV component could not be recovered by editing `TidbCluster` CR ([#3694](https://github.com/pingcap/tidb-operator/pull/3694), [@cvvz](https://github.com/cvvz))
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by dragonly — APPROVED** [review]
LGTM

**Pull request merged**